src/components/seeds/summary/AccessionByStatus.tsx

```diff
@@ -20,12 +22,13 @@ export default function AccessionByStatus({ label, status, quantity }: Props): J
       textAlign='center'
       width={isMobile ? '100%' : '20%'}
       padding='22px 10px 40px 10px'
+      onClick={() => history.push(`${APP_PATHS.ACCESSIONS}?stage=${status}`)}
```

Comment:
Link is better so we can support cmd+shift to open in a new tab.. can we move Link above the box?

Reply:
ok we can try later if it's not doable now.